NEWS.md

```diff
@@ -1,70 +1,93 @@
+# OBIC 2.3.0 2023-10-16
+## Added
+* To `crops.obic`, crop_codes: 6632 (zoete aardappelen), 6636 (naakte haver), 6660 (uien, gele, zaai), and 6664 (uien, rode, zaai)
+* Add `calc_permeability` for calculating top soil permeability 
+* Add `ind_permeability` for calculation the index for top soil permeability
+* Add vignette `obic_water_functions`, describing the functions that evaluate the effect of soil quality on groundwater quality
+
+## Changed
+* Updated N use norms for onions
+* Update calc_nleach by soil and land use dependent default n supply, issue #154
+* Update calculations in the functions `calc_pesticide_leaching`, `ind_pesticide_leaching`, `calc_psp`, `ind_psp`, `ind_gw_recharge`,`calc_n_efficiency` and `ind_n_efficiency`
+* Add soil type and land use dependent NLV estimate in calc_nleach
+
+## Fixed
+* Fixes plotting of regime curve in workability vignette #149
+* Extremely high EOS values for some cultivations fixing #162 & #134
+* Updated bibliography of packages in vignettes
+* N use norm for 1935 (maiskolfsilage)
+* N use norms of 1926 and 1927 (agrarisch natuurmengsel and overige akkerbouwgewassen), increased to match "Akkerbouwgewassen, overig" (RVO, 2022)
+
+## Deprecated
+* Building and modifying crops.obic from a script in OBIC
```

Comment:
To me it seems that this already happened in an earlier update of OBIC. The old script came along when merging the master into this branch. So this does not have to be added in the changelog